src/app/temp-converter/temp-converter.component.ts

```diff
@@ -3,13 +3,46 @@ import { Component, OnInit } from '@angular/core';
 @Component({
   selector: 'app-temp-converter',
   templateUrl: './temp-converter.component.html',
-  styleUrls: ['./temp-converter.component.css']
+  styleUrls: ['./temp-converter.component.css'],
 })
 export class TempConverterComponent implements OnInit {
+  constructor() {}
 
-  constructor() { }
-
-  ngOnInit(): void {
+  ngOnInit(): void {}
+  cel!: any;
```

Comment:
Try to mention types properly. 'any' type should be last resort. This applies for all variables.

Reply:
okay